README.md

```diff
@@ -18,7 +22,7 @@ Development requirements for PyOpia:
 3) Smaller dependency list than PySilCam -Eventual optional dependencies (e.g. for classification)
 4) Can be imported by pysilcam or other hardware-specific tools
 5) Work on a single-image basis (...primarily, with options for multiprocess to be considered later)
-6) No use of settings/config files - pass arguments directly. Eventual use of settings/config files should be handled by other repos that import PyOpia
+6) No use of settings/config files within the core code - pass arguments directly. Eventual use of settings/config files should be handled by high-level wrappers that provide arguments to functions.
 7) Github workflows
 8) Tests
```

Comment:
Maybe add a point about documentation, this might be best as point 2 (as it facilitates point 1). Just to be explicit to contributers that adding the documentation to the code is bascially mandatory for the project to succeed, and because code documentation should be compatible with readthedocs, which might not be obvious to all contributers.

Reply:
yes, good point. i have made an issue to do this